plans/software/employment_data.markdown

```diff
@@ -0,0 +1,148 @@
+# Critical Challenges Planning
+
+## The Challenge
+
+* What Critical Challenge are you going to work on?
+
+We need to effectively collect and report on employment data from recent grads and alumni.
+
+* Why is this a “challenge”?
+
+Job-hunting process is complicated involving lots of people and numerous
+steps. We don't currently have great insight into this process beyond
+word-of-mouth, individualized conversations.
+
+* Can it be solved? What would be value to the organization/team in solving it or working towards a solution?
+
+With proper tooling and great discipline around processes,
+it can probably be solved. Benefit would be having central repository of
+more objective data around job hunting. This allows us to monitor trends
+over time and is pretty important to any other challenges around job
+hunting.
+
+* What happens if we fail to solve it / improve it?
+
+We just don't know where we stand. It's impossible to accurately
+troubleshoot issues if we don't have the ability to see if our changes
+are having any impact.
+
+* What skills do you have that make you the right person to work on this challenge?
+
+Q: Good question.
+
+## The Past
+
+* What’s been done so far?
+
+- Job hunt repo
+- Word-of-mouth reports
+- Transient collection of data (slack, hand-written notes, email)
+
+* Was it effective? How do you know?
+
+Somewhat effective in the moment (for helping a single student over the
+course of a few weeks). But not very effective in aggregate or over the
+long term.
+
+* Was this someone’s challenge before you? Why did they stop doing it or how did we make progress without addressing it?
+
+- Mostly Jeff and Daisha based on tracking feedback through in-person
+  interactions.
+
+## The Present
+
+* What are the first steps that you’re ready to execute now?
+
+- Talk more to marissa, daisha, and jeff about current state of data
+  recording on job hunt -- figure out what we need and how we could most
+  easily do it
+- Research lo-fi solutions (google form, google doc, github repo) for
+  recording info
+
```

Comment:
Maybe add setting the expectation of what students should do/report after interviews and/or job offers.